build.gradle

```diff
@@ -6,7 +6,7 @@ ext {
 
   dep = [
       androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta3',
-      apolloPlugin: "com.apollographql.android:gradle-plugin:0.1.0",
+      apolloPlugin: "com.apollographql.android:gradle-plugin:0.2.0-SNAPSHOT",
```

Comment:
Is it worth having some kind of a `dev` branch that we build snapshots off instead and make that our target branch? and keep master for the releases?